tests/benchmarks/peripheral_stress_test/src/accelerometer_thread.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(accel, LOG_LEVEL_INF);
+
+#include <zephyr/drivers/sensor.h>
+#include "common.h"
+
+static const struct device *const accelerometer = DEVICE_DT_GET(DT_ALIAS(accel0));
+
+/* Accelerometer thread */
+static void accel_thread(void *arg1, void *arg2, void *arg3)
+{
+	ARG_UNUSED(arg1);
+	ARG_UNUSED(arg2);
+	ARG_UNUSED(arg3);
+
+	int ret;
+	struct sensor_value samples[3];
+	double latest[3];
+	double average[3] = {0.0};
+
+	atomic_inc(&started_threads);
+
+	if (!device_is_ready(accelerometer)) {
+		LOG_ERR("Device %s is not ready.", accelerometer->name);
+		atomic_inc(&completed_threads);
```

Comment:
Not needed here? missing increment will allow to detect that something has failed

Reply:
Return will terminate the thread, thus I increment counter of completed threads.
Failure will be detected by missing log "Accelerometer thread has completed" (line 62).